drivers/LVHDoHBASR.py

```diff
@@ -195,13 +195,25 @@ def detach(self, sr_uuid):
             except:
                 pass
 
+    def _remove_device_nodes(self):
+        """
+        Remove the kernel device nodes
+        """
+        nodes = glob.glob('/dev/disk/by-scsid/%s/*' % self.SCSIid)
+        util.SMlog('Remove_nodes, nodes are %s' % nodes)
+        for node in nodes:
+            with open('/sys/block/%s/device/delete' %
```

Comment:
If the block device is deleted, how would you recreate the SR without a host reboot?

Reply:
The SR create rescans the HBA and "rediscovers" the device.